
public class Tk259CatchingBaseAndChildExceptionClasses {

	public static void main(String[] args) {
		/*
		try
		{
			int k = 5/2;
			int myarray[] = {2,5,4};
			System.out.println(myarray[3]);
		}
		catch(Exception | ArithmeticException | ArrayIndexOutOfBoundsException e )  //you get error here ArithmeticException and ArrayIndexOutOfBoundsException are both subclasses of the Exception class. So, we get a compilation error.
		{
			System.out.println(e.getMessage());
		}
	*/
	}

}
